.github/workflows/ci_tests.yaml

```
@@ -169,6 +169,11 @@ jobs:
         path: gatekeeper
         fetch-depth: 0 # Fetch all history for all tags and branches
 
+    - name: Change test command for exempt Namespace 
+      run: |
+        replace='num_namespaces=$(($(kubectl get ns -o json | jq '\''.items | length'\'')-$(kubectl get ns | grep kube- | wc  -l)))'
```

(Optional) Slightly friendlier with double quotes; Added `^` to the regex
```suggestion
        replace="num_namespaces=$(( $(kubectl get ns -o json | jq '.items | length')-$(kubectl get ns | grep ^kube- | wc  -l) ))"
```

---

double quote interpolate  inside so `replace` gonna be just number, not "num_namespaces=$(( $(kubectl get ns -o json | jq '.items | length')-$(kubectl get ns | grep ^kube- | wc -l) )" isn't it? I tested simply
`replace="number=$(kubectl get ns)"` `echo $replace `is execution of `kubectl get ns`

---

Oh right--because it needs to be the literal command.